docs/src/pages/components/buttons/LoadingButtonsTransition.tsx

```diff
@@ -1,32 +1,30 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
 import LoadingButton from '@material-ui/lab/LoadingButton';
-import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Box from '@material-ui/core/Box';
+import MuiFormControlLabel from '@material-ui/core/FormControlLabel';
 import Switch from '@material-ui/core/Switch';
 import SaveIcon from '@material-ui/icons/Save';
 import SendIcon from '@material-ui/icons/Send';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    '& button': {
-      margin: theme.spacing(1),
-    },
-  },
-  switch: {
-    display: 'block',
-  },
-}));
+const FormControlLabel = styled(MuiFormControlLabel)(`
+  display: block;
+`);
 
 export default function LoadingButtonsTransition() {
-  const classes = useStyles();
-
   const [pending, setPending] = React.useState(false);
   function handleClick() {
     setPending(true);
   }
 
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        '& > button': {
```

Comment:
```suggestion
:not(style)
```

Reply:
You mean creating a special snare component, for demos using this format `&>: no (style)`?

Reply:
@vicasas Yes, these new styles are only meant to be temporary.

Reply:
I understand. Where in the project could I create this component? Any suggestions for the name? I can think of `BoxDemo`

Reply:
> Where in the project could I create this component?

We haven't determined yet how we will solve this problem

Reply:
I will keep it in mind